packages/braid-design-system/src/lib/components/ButtonIcon/ButtonIcon.docs.tsx

```diff
@@ -187,6 +188,53 @@ const docs: ComponentDocs = {
           </Stack>,
         ),
     },
+    {
+      label: 'Tooltip Placement',
+      background: 'surface',
+      description: (
+        <>
+          <Text>
+            Tooltips are positioned above the <Strong>ButtonIcon</Strong> by
+            default, but you can configure this via the{' '}
+            <Strong>tooltipPlacement</Strong> prop which accepts either{' '}
+            <Strong>top</Strong> or <Strong>bottom</Strong>. See{' '}
+            <TextLink href="/components/TooltipRenderer">
+              TooltipRenderer
+            </TextLink>{' '}
+            for more information.
+          </Text>
+        </>
+      ),
+      Example: () =>
+        source(
+          <Stack space="gutter">
+            <Inline space="gutter" alignY="center">
+              <ButtonIcon
+                size="standard"
+                icon={<IconArrow direction="up" />}
+                label="The tooltipPlacement is “top”"
+                id="size-1"
```

Comment:
These ids should be made unique (they are still the same as the size example)

Reply:
Oh whoops my mistake, should be fixed now